src/go/rpk/pkg/cli/cmd/container/start.go

```diff
@@ -114,12 +168,109 @@ func newStartCommand() *cobra.Command {
 	)
 	command.Flags().MarkHidden(imageFlag)
 
+	command.Flags().StringSliceVar(
+		&kafkaPorts,
+		"kafka-ports",
+		[]string{strconv.Itoa(config.DefaultKafkaPort)},
+		"List of ports that you want to use. Ports must be surely free otherwise containers won't start",
+	)
+
+	// TODO: fill in the usage strings
+	command.Flags().StringSliceVar(
+		&proxyPorts,
+		"proxy-ports",
+		[]string{"7072"},
+		"",
+	)
+	command.Flags().StringSliceVar(
+		&registryPorts,
+		"registry-ports",
+		[]string{strconv.Itoa(config.DefaultSchemaRegPort)},
+		"",
+	)
+	command.Flags().StringSliceVar(
+		&adminPorts,
+		"admin-ports",
+		[]string{strconv.Itoa(config.DefaultAdminPort)},
+		"",
+	)
+	command.Flags().StringSliceVar(
+		&brokerRPCPorts,
+		"broker-rpc-ports",
+		[]string{"33145"},
+		"",
+	)
+
 	return command
 }
 
+func assignPorts(definedPorts []string, nodesCount uint, portName string) ([]uint, error) {
+	if len(definedPorts) > 0 && len(definedPorts) > int(nodesCount) {
+		return nil, fmt.Errorf("--%s should not exceed the total number of nodes (%d)", portName, nodesCount)
+	}
+	if len(definedPorts) == 0 {
+		return nil, nil
+	}
+	ports := make([]uint, 0, len(definedPorts))
+	var any, none, withPort bool
+
+	for _, definedPort := range definedPorts {
+		switch definedPort {
+		case "none":
```

Comment:
wdyt about adding `skip` as an alias for `none` as well? I'd probably remember "skip" more myself, but "none" also seems great